kafka replay speed: add bytes limit for inflight fetch requests (#9892)
grafana/mimir: docs/sources/mimir/configure/configuration-parameters/index.md

  # CLI flag: -ingest-storage.kafka.use-compressed-bytes-as-fetch-max-bytes
  [use_compressed_bytes_as_fetch_max_bytes: <boolean> | default = true]

  # The maximum number of bytes for buffered responses to fetch requests. This

>   # limit is applied on the sum of all MaxBytes requests that are ongoing. 0 to

Reviewer: ```suggestion
  # limit applies to the sum of all ongoing MaxBytes requests. Set to 0 to
```

Reviewer: I think we can use a more active voice here.
